### ISSUE TITLE
Bug: Selected element on action bar not focused

### ISSUE THREAD

**Issue by SaiShan** [SAP/fundamental-styles#82]
#### Is this a bug, enhancement, or feature request?
Bug

#### Briefly describe your proposal.
If the element of the Action bar has been selected, the focus should be at one of those items. In the following screenshot, we cannot distinct which element was selected.

<img width="503" alt="Screen Shot 2019-06-12 at 5 36 51 PM" src="https://user-images.githubusercontent.com/25252963/59388259-54fc8b80-8d39-11e9-98e0-81587de86938.png">

**Comment by droshev** [comment]
most of the components have been refactored
